README.md

```diff
@@ -179,18 +197,19 @@ jobs:
     runs-on: ubuntu-latest
     name: Deploy
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@v3
       - name: Publish app
-        uses: cloudflare/wrangler-action@1.3.0
+        uses: cloudflare/wrangler-action@2.0.0
         with:
           apiToken: ${{ secrets.CF_API_TOKEN }}
+          command: publish
 ```
 
 If you need help defining the correct cron syntax, check out [crontab.guru](https://crontab.guru/), which provides a friendly user interface for validating your cron schedule.
 
 ### Deploying on a "dispatched" event
```

Comment:
```suggestion
### Deploying on a dispatched event
```

Reply:
I think that mentioning dispatch event here at all is an implementation concern.
Instead what the user wants to do here is trigger a deploy manually...

```suggestion
### Manually triggering a deployment
```